versioned_docs/version-1.2.x/surrealql/statements/create.mdx

```diff
@@ -25,58 +25,143 @@ CREATE [ ONLY ] @targets
+
+```surql
+-- Create a new record with a numeric id
+CREATE person:100 CONTENT {
 	name: 'Tobie',
 	company: 'SurrealDB',
 	skills: ['Rust', 'Go', 'JavaScript'],
 };
 ```
+
+## Clasues and Options
+
+Using the `ONLY` keyword, just an object for the record in question will be returned. This, instead of an array with a single object.
+
+```surql
+-- Create just a single record
+CREATE ONLY person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+```
+### Return Values
+
 By default, the create statement returns the record value once the changes have been made. To change the return value of each record, specify a `RETURN` clause, specifying either `NONE`, `BEFORE`, `AFTER`, `DIFF`, or a comma-separated list of specific fields to return.
 
 ```surql
--- Don't return any result
-CREATE person SET age = 46, username = "john-smith" RETURN NONE;
+CREATE person:20 SET age = 46, username = "john-smith" RETURN NONE;
+```
 
+```surql
 -- Return the changeset diff
-CREATE person SET age = 46, username = "john-smith" RETURN DIFF;
+CREATE person:21 SET age = 46, username = "john-smith" RETURN DIFF;
+```
 
+```surql
 -- Return the record before changes were applied
-CREATE person SET age = 46, username = "john-smith" RETURN BEFORE;
+CREATE person:22 SET age = 46, username = "john-smith" RETURN BEFORE;
+```
 
--- Return the record after changes were applied (the default)
-CREATE person SET age = 46, username = "john-smith" RETURN AFTER;
+```surql
+-- Return the record after changes were applied (the default) if any.
+CREATE person:23 SET age = 46, username = "john-smith" RETURN AFTER;
+```
 
+```surql
 -- Return a specific field only from the updated records
-CREATE person SET age = 46, username = "john-smith", interests = ['skiing', 'music'] RETURN interests;
+CREATE person:24 SET age = 46, username = "john-smith", interests = ['skiing', 'music'] RETURN interests;
 ```
+
+### Timeout
+
+The `TIMEOUT` clause can be used to specify the maximum time the statement should take to execute. This is useful when you want to limit the time a statement takes to execute.
+
+The value for `TIMEOUT` is specified in Second or milliseconds.
+
+```surql
+CREATE person:25 SET age = 46, username = "john-smith" TIMEOUT 1000ms;
+```
+
+### Parallel
+
+The `PARALLEL` keyword can be used to specify that the statement should be executed in parallel. This is useful when you want to execute a statement in parallel.
+
+```surql
+CREATE person:26, CREATE person:27 PARALLEL; 
+```
+
+## Learn more 
+
+To learn more about SurrealDB, check out the following resources: 
+- [Getting started guide](/docs/surrealdb/introduction/start)
+- [Select statement](/docs/surrealdb/surrealql/statements/select)
+- [Update statement](/docs/surrealdb/surrealql/statements/update)
+- [Insert statement](/docs/surrealdb/surrealql/statements/insert)
+- [Relate statement](/docs/surrealdb/surrealql/statements/relate)
+
```

Comment:
You basically have all the CRUD there, but not `delete` or record links. Is that intentional?

Reply:
Ah! Added that one. Need to add some logic for recommending related sections